hspec-core/src/Test/Hspec/Core/Runner.hs

```diff
@@ -368,8 +369,8 @@ runSpecForest_ config spec = do
 
   return results
 
-specToEvalForest :: Config -> [SpecTree ()] -> [EvalTree]
-specToEvalForest config =
+specToEvalForest :: Bool -> Config -> [SpecTree ()] -> [EvalTree]
+specToEvalForest useColor config =
```

Comment:
And, ya, that's where I get Weird feelings about this - a `Bool` parameter is almost always difficult to reason about unless you remember the specific implementation here.